acceptance/help_test.go

```diff
@@ -84,19 +85,20 @@ const CONFIGURE_AUTHENTICATION_USAGE = `ॐ  configure-authentication
 This unauthenticated command helps setup the internal userstore authentication mechanism for your Ops Manager.
 
 Usage: om [options] configure-authentication [<args>]
-  --client-id, -c, OM_CLIENT_ID                          string  Client ID for the Ops Manager VM (not required for unauthenticated commands)
-  --client-secret, -s, OM_CLIENT_SECRET                  string  Client Secret for the Ops Manager VM (not required for unauthenticated commands)
-  --connect-timeout, -o, OM_CONNECT_TIMEOUT              int     timeout in seconds to make TCP connections (default: 10)
-  --decryption-passphrase, -d, OM_DECRYPTION_PASSPHRASE  string  Passphrase to decrypt the installation if the Ops Manager VM has been rebooted (optional for most commands)
-  --env, -e                                              string  env file with login credentials
-  --help, -h                                             bool    prints this usage information (default: false)
-  --password, -p, OM_PASSWORD                            string  admin password for the Ops Manager VM (not required for unauthenticated commands)
-  --request-timeout, -r, OM_REQUEST_TIMEOUT              int     timeout in seconds for HTTP requests to Ops Manager (default: 1800)
-  --skip-ssl-validation, -k, OM_SKIP_SSL_VALIDATION      bool    skip ssl certificate validation during http requests (default: false)
-  --target, -t, OM_TARGET                                string  location of the Ops Manager VM
-  --trace, -tr, OM_TRACE                                 bool    prints HTTP requests and response payloads
-  --username, -u, OM_USERNAME                            string  admin username for the Ops Manager VM (not required for unauthenticated commands)
-  --version, -v                                          bool    prints the om release version (default: false)
+  --client-id, -c, OM_CLIENT_ID                            string  Client ID for the Ops Manager VM (not required for unauthenticated commands)
+  --client-secret, -s, OM_CLIENT_SECRET                    string  Client Secret for the Ops Manager VM (not required for unauthenticated commands)
+  --connect-timeout, -o, OM_CONNECT_TIMEOUT                int     timeout in seconds to make TCP connections (default: 10)
+  --decryption-passphrase, -d, OM_DECRYPTION_PASSPHRASE    string  Passphrase to decrypt the installation if the Ops Manager VM has been rebooted (optional for most commands)
+  --env, -e                                                string  env file with login credentials
+  --help, -h                                               bool    prints this usage information (default: false)
+  --ignore-verifier-warnings, OM_IGNORE_VERIFIER_WARNINGS  bool    continues working even if there are verifier warnings present (default: false)
```

Comment:
why is this a global option? `--ignore-verifier-warnings` should only be enabled for `configure-product` and `configure-director`.  A good place for this that would not affect other commands would be in the `Options` structs in `ConfigureDirector` and `ConfigureProduct`

Reply:
I made the change to scope this flag to just configure-director, since that is the only known place where a 207 is returned.